tests/integration/test_charm.py

```diff
@@ -43,3 +43,5 @@ async def test_status(ops_test):
 async def test_profile_creation(ops_test):
     run(["juju", "switch", ops_test.model_full_name])
     run(["juju", "kubectl", "apply", "-f", "./tests/integration/profile.yaml"])
+    # TODO: This does not assert anything, and may be failing in CI.  This should assert that a
+    #  namespace with the expected metadata (see label CRD) is created
```

Comment:
Is it a lot of work to add this test now?

Reply:
I'm not sure.  Might be a few hours if the odd "fails but doesn't **fail** the action" bug is non-obvious.  I'm good doing it now or tabling it for someone else while I get back to pipelines

Reply:
Let's  finish pipelines first